docs/resources/aviatrix_transit_external_device_conn.md

```diff
@@ -134,7 +134,7 @@ The following arguments are supported:
 
 ~> As of Provider version R2.21.2+, the `vpc_id` of an OCI VCN has been changed from its name to its OCID.
 * `vpc_id` - (Required) VPC ID of the Aviatrix transit gateway. For GCP BGP over LAN connection, it is in the format of "vpc_name~-~project_name".
-* `connection_name` - (Required) Transit external device connection name.
+* `connection_name` - (Optional) Transit external device connection name.
```

Comment:
Document when this is required to be set, and when it is required to be empty.

Reply:
updated